graph_pit/loss/optimized.py

```diff
@@ -82,6 +108,12 @@ class OptimizedGraphPITSourceAggregatedSDRLoss(OptimizedGraphPITLoss):
     """
     sdr_max: float = None
 
+    def __post_init__(self):
+        super().__post_init__()
+        assert self.reduction == 'sum', (
+            'Only sum reduction makes sense for SA-SDR'
+        )
```

Comment:
Can you add what the user requested in the error message?

Reply:
Done